Regression/WarpX-tests.ini

```diff
@@ -4737,3 +4737,21 @@ compileTest = 0
 doVis = 0
 compareParticles = 0
 analysisRoutine = Examples/Tests/particle_thermal_boundary/analysis_2d.py
+
+[openbc_poisson_solver]
+buildDir = .
+inputFile = Examples/Tests/openbc_poisson_solver/inputs_3d
+runtime_params = warpx.abort_on_warning_threshold = high
+dim = 3
+addToCompileString = USE_OPENPMD=TRUE USE_PSATD=TRUE
+cmakeSetupOpts = -DWarpX_DIMS=3 -DWarpX_PSATD=ON -DWarpX_OPENPMD=ON
+restartTest = 0
+useMPI = 1
+numprocs = 2
```

Comment:
@aeriforme this number (2 MPI ranks) contradicts the limitation that it only works with 1 MPI rank.

Is there an assert in the code to check this is run definitely only with 1 MPI rank?
Is there another assumption we need to check, e.g., only one box?

Reply:
The code works for more than 1 MPI rank:
- All operations (e.g., charge deposition and field gather) are done in parallel, using MPI as usual, **except** the Poisson solve
- For the Poisson solve, `rho` is gathered from all MPI rank to a single MPI rank (`ParallalCopy`), which solves the Poisson equation using non-MPI-parallelized FFT, and sends the result back to the other ranks.

Reply:
See the PR description above:
```
Also: since the solver uses a global FFT, right now only one MPI rank participates in the FFT solve (but this MPI rank can still use a full GPU) ; this could be improved later by using heFFTe. The rest of the PIC operations (charge deposition, gather, etc.) are still performed by the different MPI ranks in parallel.
```

Reply:
Thanks for clarifying!
That makes sense, I overlooked that.